main-client.js

```diff
@@ -4,6 +4,10 @@ import { createNetworkInterface } from 'apollo-client';
 import { addTypenameToSelectionSet } from 'apollo-client/queries/queryTransform';
 import { Accounts } from 'meteor/accounts-base';
 import { _ } from 'meteor/underscore';
+import { registerGqlTag } from 'apollo-client/gql';
+
+// register gql tag for queries
+registerGqlTag();
```

Comment:
Stylistically I prefer not having comments that mainly repeat the code. In this case, I'd either assume the reader knows what a gql tag is or change the comment to maybe `// setup query template literal tag`, probably former